### ISSUE TITLE
Universes and members should be locateable by relative paths.

### ISSUE THREAD

**Issue by dotsdl** [Becksteinlab/MDSynthesis#16]
At the moment, ``Universes`` and ``Members`` use only stored absolute paths for finding the files they need to generate universes and members, respectively. Relative paths from the Container's basedir are also stored, but they are not yet used. This breaks functionality for Sims and Groups when moving them around in a filesystem, even when the relative paths between these Containers and the files they require haven't changed.

The relative paths should be tried in the event the absolute path fails. If the file is found, the absolute path should be updated.

**Comment by dotsdl** [comment]
Eventually, the ``MDSynthesis.core.filesystem.Foxhound`` object will be used to sniff out missing members of Groups, but we shouldn't need to invoke it if such members can already be found by the Group with the information it already has.

**Comment by dotsdl** [comment]
Been hacking on this today, and after some significant changes to the ``GroupFile`` and ``Members`` aggregator it's become clear that a reasonable solution practically requires ``Foxhound`` to be implemented at a basic level before this issue can be resolved. I think this should happen before the 0.5.0 release, too, so the release might need pushing back.

**Comment by dotsdl** [comment]
Finally finished (mostly) an overhaul of ``Group.Members`` and ``Bundle`` that includes automatic hunting and finding of Containers that have moved. The ``Foxhound`` object handles the details of this filesystem search. After I add some explicit tests of this work I'll merge it into ``develop``.

**Comment by dotsdl** [comment]
Merged into master the work that makes ``Group`` members locateable automatically by relative paths, as well as some brief period of searching if they can't be found. Still need to include a bit of code in ``Foxhound`` to make relative paths usable by ``Sim`` to find topologies/trajectories.
